<?xml version="1.0" encoding="UTF-8"?>
<testng-results retried="6" ignored="0" total="26" passed="18" failed="2" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-08-14T15:35:20 EDT" name="Suite" finished-at="2022-08-14T15:35:58 EDT" duration-ms="37674">
    <groups>
    </groups>
    <test started-at="2022-08-14T15:35:20 EDT" name="Landing Page in Chrome" finished-at="2022-08-14T15:35:33 EDT" duration-ms="12936">
      <class name="com.qa.app.tests.LandingPageTest">
        <test-method is-config="true" signature="setUp(java.lang.String)[pri:0, instance:com.qa.app.tests.LandingPageTest@294bdeb4]" started-at="2022-08-14T15:35:20 EDT" name="setUp" finished-at="2022-08-14T15:35:28 EDT" duration-ms="8242" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[Chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method is-config="true" signature="landingPageSetUp()[pri:0, instance:com.qa.app.tests.LandingPageTest@294bdeb4]" started-at="2022-08-14T15:35:28 EDT" name="landingPageSetUp" finished-at="2022-08-14T15:35:28 EDT" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- landingPageSetUp -->
        <test-method signature="landingPageTittleTest()[pri:1, instance:com.qa.app.tests.LandingPageTest@294bdeb4]" started-at="2022-08-14T15:35:28 EDT" name="landingPageTittleTest" description="Varifying Landing Page Tittle" finished-at="2022-08-14T15:35:28 EDT" duration-ms="27" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- landingPageTittleTest -->
        <test-method signature="verifyHeader()[pri:2, instance:com.qa.app.tests.LandingPageTest@294bdeb4]" started-at="2022-08-14T15:35:28 EDT" name="verifyHeader" description="Verifying the Header on Landing Page" finished-at="2022-08-14T15:35:28 EDT" duration-ms="75" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyHeader -->
        <test-method signature="verifyLoginButton()[pri:3, instance:com.qa.app.tests.LandingPageTest@294bdeb4]" started-at="2022-08-14T15:35:28 EDT" name="verifyLoginButton" description="Verifying if Login Button is clickable or not" finished-at="2022-08-14T15:35:28 EDT" duration-ms="121" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyLoginButton -->
        <test-method signature="verifyJoinbutton()[pri:4, instance:com.qa.app.tests.LandingPageTest@294bdeb4]" started-at="2022-08-14T15:35:28 EDT" name="verifyJoinbutton" description="Verifying is Join Button is clickable or not" finished-at="2022-08-14T15:35:29 EDT" duration-ms="183" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyJoinbutton -->
        <test-method signature="getLogin()[pri:5, instance:com.qa.app.tests.LandingPageTest@294bdeb4]" started-at="2022-08-14T15:35:29 EDT" name="getLogin" description="Clicking on Login Button and verifying that user is on Login Page" finished-at="2022-08-14T15:35:31 EDT" duration-ms="2557" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getLogin -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.qa.app.tests.LandingPageTest@294bdeb4]" started-at="2022-08-14T15:35:31 EDT" name="tearDown" finished-at="2022-08-14T15:35:33 EDT" duration-ms="1671" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.qa.app.tests.LandingPageTest -->
    </test> <!-- Landing Page in Chrome -->
    <test started-at="2022-08-14T15:35:20 EDT" name="Landing Page in Edge" finished-at="2022-08-14T15:35:35 EDT" duration-ms="14904">
      <class name="com.qa.app.tests.LandingPageTest">
        <test-method is-config="true" signature="setUp(java.lang.String)[pri:0, instance:com.qa.app.tests.LandingPageTest@49096b06]" started-at="2022-08-14T15:35:20 EDT" name="setUp" finished-at="2022-08-14T15:35:31 EDT" duration-ms="11468" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[edge]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="landingPageTittleTest()[pri:1, instance:com.qa.app.tests.LandingPageTest@49096b06]" started-at="2022-08-14T15:35:31 EDT" name="landingPageTittleTest" description="Varifying Landing Page Tittle" finished-at="2022-08-14T15:35:31 EDT" duration-ms="10" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- landingPageTittleTest -->
        <test-method is-config="true" signature="landingPageSetUp()[pri:0, instance:com.qa.app.tests.LandingPageTest@49096b06]" started-at="2022-08-14T15:35:31 EDT" name="landingPageSetUp" finished-at="2022-08-14T15:35:31 EDT" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- landingPageSetUp -->
        <test-method signature="verifyHeader()[pri:2, instance:com.qa.app.tests.LandingPageTest@49096b06]" started-at="2022-08-14T15:35:31 EDT" name="verifyHeader" description="Verifying the Header on Landing Page" finished-at="2022-08-14T15:35:32 EDT" duration-ms="62" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyHeader -->
        <test-method signature="verifyLoginButton()[pri:3, instance:com.qa.app.tests.LandingPageTest@49096b06]" started-at="2022-08-14T15:35:32 EDT" name="verifyLoginButton" description="Verifying if Login Button is clickable or not" finished-at="2022-08-14T15:35:32 EDT" duration-ms="58" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyLoginButton -->
        <test-method signature="verifyJoinbutton()[pri:4, instance:com.qa.app.tests.LandingPageTest@49096b06]" started-at="2022-08-14T15:35:32 EDT" name="verifyJoinbutton" description="Verifying is Join Button is clickable or not" finished-at="2022-08-14T15:35:32 EDT" duration-ms="63" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyJoinbutton -->
        <test-method signature="getLogin()[pri:5, instance:com.qa.app.tests.LandingPageTest@49096b06]" started-at="2022-08-14T15:35:32 EDT" name="getLogin" description="Clicking on Login Button and verifying that user is on Login Page" finished-at="2022-08-14T15:35:34 EDT" duration-ms="2462" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getLogin -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.qa.app.tests.LandingPageTest@49096b06]" started-at="2022-08-14T15:35:34 EDT" name="tearDown" finished-at="2022-08-14T15:35:35 EDT" duration-ms="753" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.qa.app.tests.LandingPageTest -->
    </test> <!-- Landing Page in Edge -->
    <test started-at="2022-08-14T15:35:20 EDT" name="Login Page in Chrome" finished-at="2022-08-14T15:35:52 EDT" duration-ms="32380">
      <class name="com.qa.app.tests.LoginPageTest">
        <test-method is-config="true" signature="setUp(java.lang.String)[pri:0, instance:com.qa.app.tests.LoginPageTest@6d6cb754]" started-at="2022-08-14T15:35:20 EDT" name="setUp" finished-at="2022-08-14T15:35:29 EDT" duration-ms="8627" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[Chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method is-config="true" signature="loginPageSetUp()[pri:0, instance:com.qa.app.tests.LoginPageTest@6d6cb754]" started-at="2022-08-14T15:35:29 EDT" name="loginPageSetUp" finished-at="2022-08-14T15:35:32 EDT" duration-ms="3654" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginPageSetUp -->
        <test-method signature="verifyLoginPageHeader()[pri:1, instance:com.qa.app.tests.LoginPageTest@6d6cb754]" started-at="2022-08-14T15:35:32 EDT" name="verifyLoginPageHeader" description="Verifying the Header on Login Page" finished-at="2022-08-14T15:35:32 EDT" duration-ms="64" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyLoginPageHeader -->
        <test-method signature="verifyRememberMeLink()[pri:2, instance:com.qa.app.tests.LoginPageTest@6d6cb754]" started-at="2022-08-14T15:35:32 EDT" name="verifyRememberMeLink" description="Verifying if Remember Me link is working or not on Login Page" finished-at="2022-08-14T15:35:32 EDT" duration-ms="76" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyRememberMeLink -->
        <test-method signature="verifySignUp()[pri:3, instance:com.qa.app.tests.LoginPageTest@6d6cb754]" started-at="2022-08-14T15:35:32 EDT" name="verifySignUp" description="Verifying if Sign Up link is working or not on Login Page" finished-at="2022-08-14T15:35:32 EDT" duration-ms="66" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifySignUp -->
        <test-method signature="verifyForgotPassword()[pri:4, instance:com.qa.app.tests.LoginPageTest@6d6cb754]" started-at="2022-08-14T15:35:32 EDT" name="verifyForgotPassword" description="Verifying if Forgot Password is working or not on Login Page" finished-at="2022-08-14T15:35:33 EDT" duration-ms="79" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyForgotPassword -->
        <test-method retried="true" signature="doLogin()[pri:5, instance:com.qa.app.tests.LoginPageTest@6d6cb754]" started-at="2022-08-14T15:35:33 EDT" name="doLogin" description="Doing Login and verifying if user is Logged in or not and is on Home Page" finished-at="2022-08-14T15:35:36 EDT" duration-ms="3291" status="SKIP">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Harsh Bhut] but found []]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Harsh Bhut] but found []
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at org.testng.Assert.assertEquals(Assert.java:665)
at com.qa.app.tests.LoginPageTest.doLogin(LoginPageTest.java:47)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:962)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:806)
at org.testng.TestRunner.run(TestRunner.java:601)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:433)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:471)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:58)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- doLogin -->
        <test-method retried="true" signature="doLogin()[pri:5, instance:com.qa.app.tests.LoginPageTest@6d6cb754]" started-at="2022-08-14T15:35:36 EDT" name="doLogin" description="Doing Login and verifying if user is Logged in or not and is on Home Page" finished-at="2022-08-14T15:35:41 EDT" duration-ms="5018" status="SKIP">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at com.qa.app.pages.LoginPage.doLogin(LoginPage.java:54)
at com.qa.app.tests.LoginPageTest.doLogin(LoginPageTest.java:44)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.retryFailed(TestInvoker.java:263)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:62)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:962)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:806)
at org.testng.TestRunner.run(TestRunner.java:601)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:433)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:471)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:58)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- doLogin -->
        <test-method retried="true" signature="doLogin()[pri:5, instance:com.qa.app.tests.LoginPageTest@6d6cb754]" started-at="2022-08-14T15:35:41 EDT" name="doLogin" description="Doing Login and verifying if user is Logged in or not and is on Home Page" finished-at="2022-08-14T15:35:46 EDT" duration-ms="5019" status="SKIP">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at com.qa.app.pages.LoginPage.doLogin(LoginPage.java:54)
at com.qa.app.tests.LoginPageTest.doLogin(LoginPageTest.java:44)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.retryFailed(TestInvoker.java:263)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:62)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:962)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:806)
at org.testng.TestRunner.run(TestRunner.java:601)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:433)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:471)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:58)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- doLogin -->
        <test-method signature="doLogin()[pri:5, instance:com.qa.app.tests.LoginPageTest@6d6cb754]" started-at="2022-08-14T15:35:46 EDT" name="doLogin" description="Doing Login and verifying if user is Logged in or not and is on Home Page" finished-at="2022-08-14T15:35:51 EDT" duration-ms="5029" status="FAIL">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at com.qa.app.pages.LoginPage.doLogin(LoginPage.java:54)
at com.qa.app.tests.LoginPageTest.doLogin(LoginPageTest.java:44)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.retryFailed(TestInvoker.java:263)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:62)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:962)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:806)
at org.testng.TestRunner.run(TestRunner.java:601)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:433)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:471)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:58)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- doLogin -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.qa.app.tests.LoginPageTest@6d6cb754]" started-at="2022-08-14T15:35:52 EDT" name="tearDown" finished-at="2022-08-14T15:35:52 EDT" duration-ms="777" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.qa.app.tests.LoginPageTest -->
    </test> <!-- Login Page in Chrome -->
    <test started-at="2022-08-14T15:35:20 EDT" name="Login Page in Edge" finished-at="2022-08-14T15:35:58 EDT" duration-ms="37673">
      <class name="com.qa.app.tests.LoginPageTest">
        <test-method is-config="true" signature="setUp(java.lang.String)[pri:0, instance:com.qa.app.tests.LoginPageTest@23eee4b8]" started-at="2022-08-14T15:35:20 EDT" name="setUp" finished-at="2022-08-14T15:35:32 EDT" duration-ms="12510" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[edge]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method is-config="true" signature="loginPageSetUp()[pri:0, instance:com.qa.app.tests.LoginPageTest@23eee4b8]" started-at="2022-08-14T15:35:32 EDT" name="loginPageSetUp" finished-at="2022-08-14T15:35:35 EDT" duration-ms="2253" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginPageSetUp -->
        <test-method signature="verifyLoginPageHeader()[pri:1, instance:com.qa.app.tests.LoginPageTest@23eee4b8]" started-at="2022-08-14T15:35:35 EDT" name="verifyLoginPageHeader" description="Verifying the Header on Login Page" finished-at="2022-08-14T15:35:35 EDT" duration-ms="90" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyLoginPageHeader -->
        <test-method signature="verifyRememberMeLink()[pri:2, instance:com.qa.app.tests.LoginPageTest@23eee4b8]" started-at="2022-08-14T15:35:35 EDT" name="verifyRememberMeLink" description="Verifying if Remember Me link is working or not on Login Page" finished-at="2022-08-14T15:35:35 EDT" duration-ms="86" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyRememberMeLink -->
        <test-method signature="verifySignUp()[pri:3, instance:com.qa.app.tests.LoginPageTest@23eee4b8]" started-at="2022-08-14T15:35:35 EDT" name="verifySignUp" description="Verifying if Sign Up link is working or not on Login Page" finished-at="2022-08-14T15:35:35 EDT" duration-ms="60" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifySignUp -->
        <test-method signature="verifyForgotPassword()[pri:4, instance:com.qa.app.tests.LoginPageTest@23eee4b8]" started-at="2022-08-14T15:35:35 EDT" name="verifyForgotPassword" description="Verifying if Forgot Password is working or not on Login Page" finished-at="2022-08-14T15:35:35 EDT" duration-ms="60" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyForgotPassword -->
        <test-method retried="true" signature="doLogin()[pri:5, instance:com.qa.app.tests.LoginPageTest@23eee4b8]" started-at="2022-08-14T15:35:35 EDT" name="doLogin" description="Doing Login and verifying if user is Logged in or not and is on Home Page" finished-at="2022-08-14T15:35:40 EDT" duration-ms="5431" status="SKIP">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at com.qa.app.pages.HomePage.getName(HomePage.java:16)
at com.qa.app.tests.LoginPageTest.doLogin(LoginPageTest.java:45)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:962)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:806)
at org.testng.TestRunner.run(TestRunner.java:601)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:433)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:471)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:58)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- doLogin -->
        <test-method retried="true" signature="doLogin()[pri:5, instance:com.qa.app.tests.LoginPageTest@23eee4b8]" started-at="2022-08-14T15:35:41 EDT" name="doLogin" description="Doing Login and verifying if user is Logged in or not and is on Home Page" finished-at="2022-08-14T15:35:46 EDT" duration-ms="5417" status="SKIP">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at com.qa.app.pages.HomePage.getName(HomePage.java:16)
at com.qa.app.tests.LoginPageTest.doLogin(LoginPageTest.java:45)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.retryFailed(TestInvoker.java:263)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:62)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:962)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:806)
at org.testng.TestRunner.run(TestRunner.java:601)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:433)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:471)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:58)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- doLogin -->
        <test-method retried="true" signature="doLogin()[pri:5, instance:com.qa.app.tests.LoginPageTest@23eee4b8]" started-at="2022-08-14T15:35:46 EDT" name="doLogin" description="Doing Login and verifying if user is Logged in or not and is on Home Page" finished-at="2022-08-14T15:35:51 EDT" duration-ms="5376" status="SKIP">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at com.qa.app.pages.HomePage.getName(HomePage.java:16)
at com.qa.app.tests.LoginPageTest.doLogin(LoginPageTest.java:45)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.retryFailed(TestInvoker.java:263)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:62)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:962)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:806)
at org.testng.TestRunner.run(TestRunner.java:601)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:433)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:471)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:58)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- doLogin -->
        <test-method signature="doLogin()[pri:5, instance:com.qa.app.tests.LoginPageTest@23eee4b8]" started-at="2022-08-14T15:35:51 EDT" name="doLogin" description="Doing Login and verifying if user is Logged in or not and is on Home Page" finished-at="2022-08-14T15:35:57 EDT" duration-ms="5389" status="FAIL">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at com.qa.app.pages.HomePage.getName(HomePage.java:16)
at com.qa.app.tests.LoginPageTest.doLogin(LoginPageTest.java:45)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.retryFailed(TestInvoker.java:263)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:62)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:962)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:806)
at org.testng.TestRunner.run(TestRunner.java:601)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:433)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:471)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:58)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- doLogin -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.qa.app.tests.LoginPageTest@23eee4b8]" started-at="2022-08-14T15:35:57 EDT" name="tearDown" finished-at="2022-08-14T15:35:58 EDT" duration-ms="675" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.qa.app.tests.LoginPageTest -->
    </test> <!-- Login Page in Edge -->
  </suite> <!-- Suite -->
</testng-results>
